database/migrations/2018_06_04_128800_create_app_history_table.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateAPPHISTORYTable extends Migration
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_HISTORY', function (Blueprint $table) {
```

Comment:
Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.